Classes/Command/ResultPrinter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace OliverKlee\Phpunit\Command;
+
+use PHPUnit_Framework_Exception;
+use PHPUnit_TextUI_ResultPrinter;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Result printer for Symfony console command.
+ *
+ * @author Felix Semmler <felix.semmler@aoe.com>
```

Comment:
We don't adds authors to files anymore, haven't done that for ages. I always removes it when seeing it.